src/simulation/m_rhs.fpp

```diff
@@ -786,8 +789,6 @@ contains
             irx%end = m; iry%end = n; irz%end = p
             ! ===============================================================
 
-            call nvtxStartRange("RHS-RIEMANN-SOLVER")
```

Comment:
please use the all-caps notation like what was there before...

Reply:
Resolved